lib/config/options/index.ts

```diff
@@ -1218,6 +1218,7 @@ const options: RenovateOptions[] = [
     description:
       'If set to `false`, Renovate will upgrade dependencies to their latest release only. Renovate will not separate major or minor branches.',
     type: 'boolean',
+    default: true,
```

Comment:
Do we need to create a new issue to "check the code for more logical errors like this"? It feels like you found a new category or class or bug here.